### ISSUE TITLE
screen reader jaws issue with code examples in quiz

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/freeCodeCamp#60493]
We have this report from the forum here 

https://forum.freecodecamp.org/t/screen-reader-jaws-issue-with-code-examples-for-quiz/748155

They are saying that when they try to do a quiz and there is a code example, the devices they are using don't always read the code. This puts them in a situation where they don't know how to answer the question and just guess.

We need to test this on our end and come up with a solution so everyone can take the quizzes and pass

Here are a couple of quizzes with code examples to test this out on

https://www.freecodecamp.org/learn/full-stack-developer/quiz-basic-css/quiz-basic-css

https://www.freecodecamp.org/learn/full-stack-developer/quiz-css-positioning/quiz-css-positioning

**Comment by jdwilkin4** [comment]
I was able to test this out with VoiceOver and here is what I found.

When using the screen reader when I try to select a code example, it just says the following:

<img width="929" alt="Image" src="https://github.com/user-attachments/assets/91769b78-f8c7-491c-bab6-affb65b622c1" />

This is in line with what the camper is experiencing. 

So we need to find a way to get the screen reader to actually read the code.

It looks like we are using the UI quiz components here

https://github.com/freeCodeCamp/ui/blob/main/src/quiz-question/quiz-question.tsx

We are using the  `RadioGroup` component from Headless UI. So we will need to do some research on how to get this to be more screen reader friendly.

**Comment by jdwilkin4** [comment]
When I open up the DevTools, the code example options have a bunch of nested spans.

<img width="993" alt="Image" src="https://github.com/user-attachments/assets/82494528-aa9e-4b36-9e55-0b51e3a796b0" />

 So we will need to find a way to just get the screen reader to read out the raw code and nothing else.

**Comment by blindgeekzone** [comment]
hi, okay, i know this will take time. hope a solution can be found. will try to be patient. but dont want to be waiting weeks and cannot move on in the order or the course. so this is the last step in the forms and tables. before moving on.
marvin..

**Comment by blindgeekzone** [comment]
Hi. So did this issue get rectified.

Marvin.

**Comment by majestic-owl448** [comment]
The issue is solved, the fix will be live after next deployment

**Comment by blindgeekzone** [comment]
Hi. Thanks for that. When is the next deployment.

Marvin.

**Comment by huyenltnguyen** [comment]
The fix has been deployed and it is now live.
